pkg/collector/corechecks/cluster/orchestrator/processors/processor.go

```diff
@@ -83,6 +87,12 @@ type Processor struct {
 	h Handlers
 }
 
+// ProcessResult contains the processing result of metadata and manifest
+type ProcessResult struct {
+	MetadataMessages []model.MessageBody
```

Comment:
same comment as before regarding the different.
Alternative: you could map them to types to make the distinction more clear